ui/lib/pki/addon/components/page/pki-issuer-rotate-root.hbs

```diff
@@ -169,14 +167,10 @@
       </div>
 
       <div class="field is-grouped box is-fullwidth is-bottomless">
-        <div class="control">
-          <button type="submit" class="button is-primary" data-test-pki-rotate-root-save>
-            Done
-          </button>
-          <button {{on "click" @onCancel}} type="button" class="button has-left-margin-s" data-test-pki-rotate-root-cancel>
-            Cancel
-          </button>
-        </div>
+        <Hds::ButtonSet>
```

Comment:
same comment here above if you want to cleanup the `<div>` on line 169 and replace with `<hr class="has-background-gray-100" />`